pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,9 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 {
+				obj.Termination = TLSTerminationEdge
```

Comment:
> Yes, it does. Actually I am not sure we should set any default for this. @smarterclayton how do you envision defaulting here?

Basically, we have a problem where an existing object has a bunch of ignored data set, but there's no obvious indication that your intent (secure my route) failed.  The entire stanza has no reason to exist if its not used, so we probably should always have rejected it.

Dropping the entire stanza would maintain current behavior and allow us to tighten validation, but it would drop some (unused) data out of the object.  @smarterclayton would that satisfy?